script/bootstrap

```diff
@@ -1,9 +1,10 @@
 #!/usr/bin/env bash
 set -e
 
-# Only set GOPATH on non-Windows platforms as Windows can't do the symlinking
-# that we need.
-if [ -z "$GOPATH" ] && uname -s | grep -vq "_NT-"; then
+# Tell MSYS to really create symbolic links.
+export MSYS=winsymlinks:nativestrict
```

Comment:
Ah, good idea. If we're in a Windows environment while running `script/bootstrap`, _then_ we can set `$MSYS`.